cartridges/int_stickyio_sfra/cartridge/controllers/CheckoutServices.js

```diff
@@ -23,18 +23,21 @@ if (stickyioEnabled) {
         server.middleware.https,
         csrfProtection.validateAjaxRequest,
         function (req, res, next) {
+        	var BasketMgr = require('dw/order/BasketMgr');
```

Comment:
Please correct spacing issues in line 26?

Reply:
fixed